app/src/main/java/com/example/taskyy/domain/navigation/Navigation.kt

```diff
@@ -107,29 +112,82 @@ fun Nav() {
             composable(route = Screen.Reminder.route + "/{dateString}") {
                 val reminderViewModel = hiltViewModel<ReminderViewModel>()
                 val state by reminderViewModel.state.collectAsState()
+                val editedRemindDescription = navController
+                    .currentBackStackEntry
+                    ?.savedStateHandle
+                    ?.getStateFlow<String>("reminderDescription", "Reminder Description")
+                    ?.collectAsStateWithLifecycle()
+
+                LaunchedEffect(editedRemindDescription) {
+                    reminderViewModel.setReminderDescription(
+                        editedRemindDescription?.value ?: "Reminder Description"
+                    )
+                }
+
                 ReminderScreen(
-                    state = state,
-                    onEvent = { event ->
-                        when (event) {
-                            is ReminderEvent.EnterReminderDescription -> navController.navigate(
-                                Screen.ReminderDetails.route
+                    state = state
+                ) { event ->
+                    when (event) {
+                        is ReminderEvent.EnterReminderDescription -> {
+                            navController.previousBackStackEntry?.savedStateHandle?.set(
+                                "screenType",
+                                event.editDescription
                             )
+                            navController.navigate(Screen.EditText.route)
```

Comment:
Navigating to a screen with parameter (which type of edit text field it is) still works with normal Compose navigation arguments, but here you don't pass any to that navigation. The route should look something like this:
```
"edit_text/{screenType}
```
And then navigating to it like this:
```
navController.navigate("edit_text/${ScreenType.TITLE}")
```